RELEASING.md

```diff
@@ -62,12 +86,26 @@ What is SCM release tag or label for "ActiveMQ Artemis Parent"? (org.apache.acti
 What is the new development version for "ActiveMQ Artemis Parent"? (org.apache.activemq:artemis-pom) 1.4.1-SNAPSHOT: : 1.5.0-SNAPSHOT
 ```
 
-Otherwise snapshots will be created at 1.4.1 and forgotten. (Unless we ever elease 1.4.1 on that example).
+Otherwise snapshots will be created at 1.4.1 and forgotten. (Unless we ever release 1.4.1 on that example).
 
 For more information look at the prepare plugin:
 
 - http://maven.apache.org/maven-release/maven-release-plugin/prepare-mojo.html#pushChanges
 
+If you set `pushChanges=false` then you will have to push the changes manually.  The first command is to push the commits
+which are for changing the `&lt;version>` in the pom.xml files, and the second push is for the tag, e.g.:
+
+```sh
+git push upstream
+git push upstream <version>
+```
+
+
+## Extra tests
+
+Note: The Apache Release plugin does not bump the version on the `extraTests` module.  Release manager should manually
```

Comment:
Perhaps the poms should be fixed so that it does. I presume its skipped over because the module is not actually enabled at the time, so perhaps it can be updated to be enabled during the release and allow it to be updated, without actually doing anything else (e.g it needn't be deployed etc).